hubspot/connection/__init__.py

```diff
@@ -204,6 +204,7 @@ def _require_successful_response(response):
             raise exception_class(
                 error_data['message'],
                 error_data['requestId'],
+                error_data,
```

Comment:
Hi @LuRsT, I don't think that the detailed contents of error_data are documented. As an example, `failureMessages` isn't mentioned on the [page that documents the resource that returns it](http://developers.hubspot.com/docs/methods/contacts/batch_create_or_update).

Without such documentation I'm not sure how you would compile a list of all of the error possibilities. You could try to do it by experimenting with the API, but there'd always be the possibility that a user of the library might come across an error situation (and corresponding HubSpot response) that your experiments hadn't uncovered.